pulpcore/pulpcore/app/models/publication.py

```diff
@@ -20,6 +20,8 @@ class Publication(Model):
 
     repository_version = models.ForeignKey('RepositoryVersion', on_delete=models.CASCADE)
 
+    complete = models.BooleanField(default=False)
```

Comment:
An unwritten convention has been to group the fields together followed by the relations (grouped).  For consistency, please move the `complete` field to be with `created` and add to `Fields` in the docstring.

Reply:
will do.